src/Nancy/DefaultRootPathProvider.cs

```diff
@@ -14,7 +14,7 @@ public class DefaultRootPathProvider : IRootPathProvider
         public string GetRootPath()
         {
 #if CORE
-            return Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationBasePath;
+            return System.IO.Directory.GetCurrentDirectory();

```

Comment:
Not to mention that the behavior would diverge between netcore and full-framework